src/views/common/TimeScrollView/TimeScrollView.tsx

```diff
@@ -78,17 +78,16 @@ const TimeScrollView = <T extends {[key: string]: any}> (props: TimeScrollViewPr
 
     // TODO: It'd be nice to show some sort of visual indication of how much zoom has been requested,
 
-    const content = (
-        <div
-            style={{width: width - toolbarWidth, height, position: 'relative'}}
-            onWheel={handleWheel}
-            onMouseDown={handleMouseDown}
-            onMouseUp={handleMouseUp}
-            onMouseMove={handleMouseMove}
-            onMouseOut={handleMouseLeave}
-        >
+    const effectiveWidth = useMemo(() => width - toolbarWidth, [width, toolbarWidth])
```

Comment:
This computation is referenced repeatedly, so I broke it out into its own thing.